generate-sources/new-plugin/src/main/kotlin/io/arrowkt/example/TransformNewSources.kt

```diff
@@ -20,6 +20,7 @@ import arrow.meta.quotes.classDeclaration
 
   If base directory is not provided, it will be "build" directory in the Gradle daemon workspace.
  */
+/* TODO 58: rewrite
 val Meta.transformNewSources: CliPlugin
```

Comment:
Should I rewrite existing case, using `IR`?

Reply:
We can but this example name is misleading since your are no longer "transforming new sources" Instead we are transforming generated IR.